### PR TITLE
reformatted to use jira markdown and refer to jira

### PR DESCRIPTION
**Pull request by timothy-spencer**
I mainly used markdown2confluence to fix this up, then did some tweaks by hand.

### CODE REVIEW

File: OnboardingChecklist.md [hunk 1]
```diff
@@ -4,219 +4,230 @@
 
 When someone new joins the cloud.gov team:
 
-1. Create a new issue in `cg-product` called "Ramp up [person's name] on cloud.gov team".
+0. Get a Jira account for NewPerson.  Buddy can help, or you can go to the Program Manager.
+1. Create a new issue in Jira under the `Product Management` board called "Ramp up [NewPerson] on cloud.gov team".
```
Comment (mogul) [inline OnboardingChecklist.md]: s/under the `Product Management` board/in the cloud.gov (CG) project assigned to the `Product Management` team/

File: OnboardingChecklist.md [hunk 2]
```diff
@@ -4,219 +4,230 @@
...
 2. View the raw source of this file.
 3. Copy everything below the line into the new issue's body.
 4. Replace "NewPerson" with the new person's name.
 5. Replace "Buddy" with the onboarding buddy's name.
 4. Delete any checklists irrelevant for the new person's skill domain (theme).
 5. Submit the issue.
 6. Assign the issue to the person who bravely volunteered to be the new person's Onboarding Buddy.
-7. Put the issue into the _In Progress_ pipeline in Favro on the _cloud.gov
-   Program_ **Program Board**.
+7. Put the issue into the _In Progress_ lane in Jira on the cloud.gov *Program Board*.
```
Comment (mogul) [inline OnboardingChecklist.md]: s/lane/column/
s/Program Board/Program Management Board/

File: OnboardingChecklist.md [hunk 3]
```diff
@@ -4,219 +4,230 @@
...
+* [ ] Read through [the Overview section of cloud.gov|https://cloud.gov/overview/] for a broader understanding of cloud.gov, especially as we present it to potential customers/users.
+* [ ] Read the [January 2016 cloud.gov &quot;Executive Business Case&quot; document|https://docs.google.com/document/d/138OcG0Lt6gr9J0wM0TzzPNyTROmYAwfLIDujtweiwGw/edit#] for greater context about cloud.gov&#39;s potential impact in government.
+* [ ] [Deploy a sample application to cloud.gov|https://cloud.gov/docs/getting-started/your-first-deploy/] to get familiar with the basics of the PaaS from a user&#39;s perspective.
+* [ ] Sign up for a [Favro account|https://favro.com/signup] using your GSA Google Account. Add a profile photo and add your name/Github handle using this standard so you receive notifications on any linked Github issues: {noformat}[Github handle] - [your name]{noformat}, e.g. {noformat}suprenant - Andrew Suprenant{noformat}.
```
Comment (mogul) [inline OnboardingChecklist.md]: We should strike this line since Favro's no longer in the picture.

File: OnboardingChecklist.md [hunk 4]
```diff
@@ -4,219 +4,230 @@
...
+* [ ] Check out the [Favro cloud.gov Program
+* board|https://favro.com/organization/1e11108a2da81e3bd7153a7a/0b64f44bc57f65052fad8244] and make sure you have access to all our Favro boards.
+* [ ] Check out the [roadmap|https://favro.com/organization/1e11108a2da81e3bd7153a7a/0b64f44bc57f65052fad8244] to get a high-level view of recently-completed, in-progress, and upcoming features.
```
Comment (mogul) [inline OnboardingChecklist.md]: This one will need editing, but I don't know a suitable replacement link yet... Maybe just note this is out of date?

File: OnboardingChecklist.md [hunk 5]
```diff
@@ -4,219 +4,230 @@
...
+h3. Theme-specific items
+
+For explanations of our theme names, see [this glossary|https://github.com/18F/cg-product#sub-teamsthemes-of-work].
+
+h4. Platform-specific items
+
+* [ ] Review the System Security Plan (the latest version lives on [Google Drive|https://drive.google.com/drive/u/0/folders/0B6fPl5s12igNX3JwR2xFZVpmek0]; look for &quot;cloud.gov System Security Plan (SSP)&quot; as a {{.docx}} file). Of particular note for onboarding:
+* Section 9: System Description
+** Section 10: System Environment
+* [ ] Bookmark [the Platform kanban board view|https://favro.com/organization/1e11108a2da81e3bd7153a7a/68a186775e1a0ee297ee81ed].
```
Comment (mogul) [inline OnboardingChecklist.md]: This should point to https://cm-jira.usa.gov/secure/RapidBoard.jspa?rapidView=1926&view=detail&selectedIssue=CG-107

File: OnboardingChecklist.md [hunk 6]
```diff
@@ -4,219 +4,230 @@
...
+* Get familiar with our AWS setup
+** [ ] Read about [AWS onboarding|https://cloud.gov/docs/ops/aws-onboarding/]
+** [ ] Read about the [AWS CLI|https://aws.amazon.com/cli/]
+* [ ] Review the [daily maintenance tasks|https://cloud.gov/docs/ops/maintenance-list/] for when you&#39;re on support duty
+
+h4. Customer-specific items
+
+* [ ] Join the {{#cg-customer}} channel on Slack.
+* [ ] Ping @standup-bot for instructions on front end channel standup.
+* [ ] Review the [Customer board|https://favro.com/organization/1e11108a2da81e3bd7153a7a/e9acfea577acd5bdadf3d6a2].
```
Comment (mogul) [inline OnboardingChecklist.md]: Should be https://cm-jira.usa.gov/secure/RapidBoard.jspa?rapidView=1929&view=detail&selectedIssue=CG-360

File: OnboardingChecklist.md [hunk 7]
```diff
@@ -4,219 +4,230 @@
...
+* [ ] Identify a straightforward, well-groomed story in progress that involves their skills domain, schedule a meeting with the owner for an introduction (if it&#39;s not you), and setup pairing sessions several times in the first week on the project.
+* [ ] Identify a straightforward, well-groomed first story, ideally something they could conceivably complete in their first two/three weeks using their existing skills. Discuss the context with them, then make them the assignee for the card.
+* [ ] Discuss suggestions for how the onboarding experience could have been improved and open as PRs on [the onboarding template|https://github.com/18F/cg-product/blob/master/OnboardingChecklist.md].
+* [ ] Ask the Program Manager or Director to add the person to [Zendesk|https://cloud-gov.zendesk.com], so they can see how we handle non-18F support and read technical discussions happening with outside groups.
+* [ ] Invite them to the private Slack channel cg-supportstream, used for backchanneling on support interactions.
+* [ ] Ask the Program Manager or Director to add them to the 18F organization in Favro.
```
Comment (mogul) [inline OnboardingChecklist.md]: Strike this one; not needed as adding someone to Jira will result in them having the right perms.

**Comment by timothy-spencer** [comment]
Hmm.  Not sure why quoting is escaped.  That was done by the markdown2confluence utility.  Removed that.

**Pull request merged**